go.mod

```diff
@@ -30,7 +29,7 @@ require (
 replace (
 	github.com/lightninglabs/lndclient => github.com/lightninglabs/lndclient v1.0.1-0.20200708223031-76709c25d859
 	github.com/lightninglabs/loop => github.com/lightninglabs/loop v0.6.5-beta
-	github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.10.3-beta
+	github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.11.0-beta
```

Comment:
I don't think we can update `lnd` and `loop` in two separate PRs (speaking about #104).
Everything's really entangled with the `lndclient` library. Took me a while to find the correct commits/versions for all 4 projects (lnd, lndclient, loop, faraday).
Here's what ended up working for me (notice the `replace` directives that are now removed):
```
module github.com/lightninglabs/lightning-terminal

require (
	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
	github.com/btcsuite/btcwallet v0.11.1-0.20200814001439-1d31f4ea6fc5 // indirect
	github.com/desertbit/timer v0.0.0-20180107155436-c41aec40b27f // indirect
	github.com/grpc-ecosystem/grpc-gateway v1.14.3
	github.com/improbable-eng/grpc-web v0.12.0
	github.com/jessevdk/go-flags v1.4.0
	github.com/lightninglabs/faraday v0.2.0-alpha.0.20200811090810-93547d433a66
	github.com/lightninglabs/lndclient v1.0.1-0.20200629081038-bb0726595df9
	github.com/lightninglabs/loop v0.8.0-beta
	github.com/lightningnetwork/lnd v0.11.0-beta.rc2
	github.com/lightningnetwork/lnd/cert v1.0.2
	github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
	github.com/mwitkow/grpc-proxy v0.0.0-20181017164139-0f1106ef9c76
	github.com/prometheus/client_golang v1.5.1 // indirect
	github.com/rakyll/statik v0.1.7
	github.com/rs/cors v1.7.0 // indirect
	golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
	golang.org/x/sys v0.0.0-20200406155108-e3b113bbe6a4 // indirect
	google.golang.org/grpc v1.28.0
	gopkg.in/macaroon-bakery.v2 v2.1.0
	gopkg.in/macaroon.v2 v2.1.0
)

go 1.13
```

Reply:
Thanks, this is good to know. When I did them separately, I didn't get any build errors and the app appeared to run fine in my manual testing. I will merge the PRs and use the `go.mod` above. 

I'm curious, how can I determine if there are incompatibilities in the future with these dependencies? Also, why did you use the RC2 of `lnd v0.11.0` instead of the final release?

Reply:
Discussed offline a bit. To answer the question about dependencies: Usually there shouldn't be any `replace` directives in there unless there's a good and specific reason. So removing those and not getting any build errors is usually a good indication that you're on the right track.